### ISSUE TITLE
backend/s3: Support New eu-north-1 Region Automatic Validation

### ISSUE THREAD

**Issue by bflad** [hashicorp/terraform#19632]
### Current Terraform Version

```
terraform 0.11.10
```

### Use-cases

AWS has just publicly announced the availability of the `eu-north-1` (Stockholm) region: https://aws.amazon.com/blogs/aws/now-open-aws-europe-stockholm-region/

Automatic region validation occurs from the AWS Go SDK provided default endpoints file, which added support in v1.16.3.

### Attempted Solutions

Before the updated validation is released, it should be possible to workaround this by [skipping region validation](https://www.terraform.io/docs/backends/types/s3.html#skip_region_validation):

```hcl
terraform {
  backend "s3" {
    # ... potentially other configuration ...
    region                 = "eu-north-1"
    skip_region_validation = true
  }
}
```

### Proposal

Update the following dependencies to automatically pick up the change:
* aws/aws-sdk-go@v1.16.3
* terraform-providers/terraform-provider-aws@v1.52.0 (releasing later today, hopefully)

### References

* https://github.com/terraform-providers/terraform-provider-aws/issues/6818
* https://aws.amazon.com/blogs/aws/now-open-aws-europe-stockholm-region/

**Comment by apparentlymart** [comment]
:+1: feel free to PR this when the other parts are in place!

**Comment by bflad** [comment]
Support for this has been merged in and will release in 0.12.0-alpha5 or 0.12.0-beta1, whichever is the next release. 👍

**Comment by Subham9418** [comment]
> Support for this has been merged in and will release in 0.12.0-alpha5 or 0.12.0-beta1, whichever is the next release. 👍

Is there any ETA for this?
